rds/security_group.tf

```diff
@@ -4,16 +4,28 @@ resource "aws_security_group" "sg_rds" {
   description = "Security group that is needed for the RDS"
   vpc_id      = "${var.vpc_id}"
 
-  ingress {
-    from_port       = "${lookup(var.ports, var.rds_type)}"
-    to_port         = "${lookup(var.ports, var.rds_type)}"
-    protocol        = "tcp"
-    security_groups = ["${var.security_groups}"]
-  }
-
   tags {
     Name        = "${var.project}-${var.environment}${var.tag}-sg_rds"
     Environment = "${var.environment}"
     Project     = "${var.project}"
   }
 }
+
+resource "aws_security_group_rule" "rds_sg_in" {
+  count                    = "${length(var.security_groups)}"
```

Comment:
yep it's working here: https://github.com/skyscrapers/skyscrapers-terraform/blob/master/stacks/test/static/concourse_k8s_rds.tf#L24